services/app-api/forms/sar.json

```diff
@@ -14,10 +14,143 @@
       "verbiage": {
         "intro": {
           "section": "",
-          "subsection": "",
-          "hint": "",
-          "exportSection": ""
+          "subsection": "General Information",
+          "hint": ""
```

Comment:
I don't see `hint` in [types](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/main/services/ui-src/src/types/reports.ts#L16-L24) (I linked main, not your branch, but it's not in either)

It does render though 🤷